src/PWFusion_MAX31856.h

```diff
@@ -66,6 +66,22 @@ typedef enum Max31856_Reg_e {
   REG_SR      = 0x0F  // Status Register
 } Max31856_Reg;
 
+typedef enum {

```

Comment:
Would you add these new enum values to keywords.txt in the root folder of the project?   The 1.x Arduino IDE has pretty terrible syntax highlighting and requires you to list all the keywords for it to highlight.

Reply:
Fixed in 1c0e667cec8ff3531e8d1644f32b13843cc2e90e

Reply:
This is stupid, but...   Can you try using the library with a 1.x IDE.   I think you have to have exactly one tab between keywords and values.    If it works as is, go ahead and merge this

Reply:
Nope, you're right, strait from Arduino CLI docs:

> keywords.txt is formatted in four fields which are separated by a single true tab (not spaces):

I'll get it fixed

Reply:
Converted to proper syntax in 19b5db4